src/AzureExtension/Assets/QuickStartPlayground/docsEmbeddings-OpenAI.json

```diff
@@ -1549,7 +1549,7 @@
+        "code": "\n\n=== ./main.cpp ===\n#include <iostream>\n#include <string>\n\nusing namespace std;\n\nint main()\n{\n    cout << \"Hello World\\n\";\n    cout << \"Input: \";\n    string data;\n    getline(cin, data);\n    cout << \"Output: \" << data << \"\\n\\n\";\n    return 0;\n}\n\n=== ./.vscode/launch.json ===\n{\n    \"version\": \"0.2.0\",\n    \"configurations\": [\n        {\n            \"name\": \"Launch Main\",\n            \"type\": \"cppdbg\",\n            \"request\": \"launch\",\n            \"program\": \"${workspaceFolder}/main.out\",\n            \"args\": [],\n            \"stopAtEntry\": false,\n            \"cwd\": \"${workspaceFolder}\",\n            \"environment\": [],\n            \"externalConsole\": false,\n            \"MIMode\": \"gdb\",\n            \"setupCommands\": [\n                {\n                    \"description\": \"Enable pretty-printing for gdb\",\n                    \"text\": \"-enable-pretty-printing\",\n                    \"ignoreFailures\": true\n                }\n            ],\n            \"preLaunchTask\": \"Build Main\"\n        }\n    ]\n}\n\n=== ./.vscode/tasks.json ===\n{\n    \"version\": \"2.0.0\",\n    \"tasks\": [\n        {\n            \"label\": \"Build Main\",\n            \"type\": \"shell\",\n            \"command\": \"g++ -g main.cpp -o main.out\",\n            \"group\": {\n                \"kind\": \"build\",\n                \"isDefault\": true\n            }\n        }\n    ]\n}\n",

```

Comment:
The way this file is structured doesn't lend itself to diffing- but the changes here are to add an additional `\n` to the very end of the section (for consistency with other examples), and to remove the `prompt.txt` content (which is from an earlier version of Quickstart and no longer used).